src/overlays/actors/ovl_Dm_Tag/z_dm_tag.c

```diff
@@ -31,30 +30,240 @@ const ActorInit Dm_Tag_InitVars = {
+    s32 pad;
+    Actor* sp30;
+    Actor* sp2C;
+    s16 sp2A = this->actor.cutscene;
+    s32 ret = false;
+
+    sp30 = func_80C22350(this, play, ACTORCAT_NPC, ACTOR_EN_AN);
+    sp2C = func_80C22350(this, play, ACTORCAT_NPC, ACTOR_EN_AH);
+
+    switch (this->unk_1A4) {
+        case 0:
+            if (func_80C22400(this, sp2A)) {
+                if ((sp2C != NULL) && (sp2C->update != NULL)) {
+                    Camera_SetTargetActor(Play_GetCamera(play, ActorCutscene_GetCurrentCamera(sp2A)), sp2C);
+                }
+                this->unk_1A4++;
+                ret = true;
+            }
+            break;
+
+        case 1:
+            if ((sp30 != NULL) && (sp30->update != NULL)) {
+                Camera_SetTargetActor(Play_GetCamera(play, ActorCutscene_GetCurrentCamera(func_80C2247C(this, 0))),
+                                      sp30);
+            }
+            this->unk_1A4++;
+            ret = true;
+            break;
+
+        case 2:
+            ActorCutscene_Stop(func_80C2247C(this, 0));
+            if (func_80C22400(this, func_80C2247C(this, 1))) {
+                this->unk_1A4++;
+                ret = true;
+            }
+            break;
+
+        case 3:
+            ActorCutscene_Stop(func_80C2247C(this, 1));
+            if (func_80C22400(this, func_80C2247C(this, 2))) {
+                this->unk_1A4++;
+                ret = true;
+            }
+            break;
+
+        case 4:
+            ActorCutscene_Stop(func_80C2247C(this, 2));
+            if (func_80C22400(this, func_80C2247C(this, 3))) {
+                this->unk_1A4++;
+                ret = true;
+            }
+            break;
+
+        case 5:
+            ActorCutscene_Stop(func_80C2247C(this, 3));
+            if (func_80C22400(this, func_80C2247C(this, 4))) {
+                this->unk_1A4++;
+                ret = true;
+            }
+            break;
+
+        case 6:
+            func_800B7298(play, &this->actor, 7);
+            play->nextEntranceIndex = 0xBC50;
+            gSaveContext.nextCutsceneIndex = 0;
+            play->sceneLoadFlag = 0x14;
+            play->unk_1887F = 2;
+            gSaveContext.nextTransition = 6;
+            this->unk_1A4++;
+            break;
+    }
+    return ret;
+}
+
+s32 func_80C227E8(DmTag* this, PlayState* play) {
+    if (this->unk_1A4 == 0) {
+        func_800B7298(play, &this->actor, 7);
+        play->nextEntranceIndex = 0xBC40;
+        gSaveContext.nextCutsceneIndex = 0;
+        play->sceneLoadFlag = 0x14;
+        play->unk_1887F = 2;
+        gSaveContext.nextTransition = 6;
+        this->unk_1A4++;
+    }
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Tag/func_80C22350.s")
+s32* func_80C22880(DmTag* this, PlayState* play) {
+    s32 time;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Tag/func_80C22400.s")
+    switch (this->unk_18E) {
+        case 1:
+            time = gSaveContext.save.time - 0x3FFC;
```

Comment:
This looks like `SCHEDULE_TIME_NOW` but doesn't use the schedule. To be honest I don't know what to make of it.